repository/s.json

```diff
@@ -1966,6 +1966,15 @@
 				}
 			]
 		},
+		{
+			"details": "https://github.com/zharkomi/SimpleIndent",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "main"
```

Comment:
Branch based releases are not supported anymore. Please refer to the docs, check off the tasks in the PR template, and report back.

Reply:
You might also want to specify labels.